galaxy_ng/app/api/ui/viewsets/collection.py

```diff
@@ -314,7 +315,7 @@ def retrieve(self, request, *args, **kwargs):
 
 
 class CollectionRemoteViewSet(api_base.ModelViewSet):
-    queryset = CollectionRemote.objects.filter(name__in=['rh-certified', 'community'])
```

Comment:
I think it might be a better idea to remove the filter here. We don't allow people to add new remotes, so this hopefully won't be needed.